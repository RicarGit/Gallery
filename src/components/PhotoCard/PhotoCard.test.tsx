import { render } from '@testing-library/react'
import { PhotoCard } from '.'

describe('PhotoCard', () => {
  it('should match snapshot', () => {
    const { container } = render(
      <PhotoCard url='www.google.com' name='image-name' photos={[]} setPhotos={() => []} />
    )

    expect(container.firstChild).toMatchSnapshot()
  })
})