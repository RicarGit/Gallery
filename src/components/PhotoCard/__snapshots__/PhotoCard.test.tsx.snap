// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PhotoCard should match snapshot 1`] = `
<div
  class="PhotoCardstyled__Container-sc-mpou7f-0 guwiQZ"
>
  <img
    alt="Image-name"
    class="cardImage"
    src="www.google.com"
  />
  <p>
    Image-name
  </p>
  <div
    class="Downloadstyled__Container-sc-9k28x7-0 ezUDzu"
  >
    <div
      class="download"
    >
      <img
        alt="download icon"
        src="download.svg"
      />
    </div>
  </div>
  <div
    class="TrashCanstyled__TrashCan-sc-i65z8w-0 eBYaF"
  >
    <img
      alt="trash icon"
      class="trashCan"
      src="delete.svg"
    />
  </div>
</div>
`;
